src/Symfony/Bridge/PhpUnit/bin/simple-phpunit.php

```diff
@@ -78,11 +78,11 @@
     $PHP .= ' -qrr';
 }
 
-$defaultEnvs = array(
+$defaultEnvs = [
```

Comment:
please revert all these changes, they are false positive on the phpunit-bridge

Reply:
done